go.mod

```diff
@@ -1,10 +1,12 @@
 module oras.land/oras-go
 
-go 1.22
+go 1.22.4
+
+toolchain go1.22.5
```

Comment:
Is this line required?

Reply:
It's generated by `go mod tidy`. Let me try to remove it and see what happens.

Reply:
OK, I removed the toolchain and the build still passed.